jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListPresenter.java

```diff
@@ -321,10 +298,12 @@ private ProcessInstanceSummary createProcessInstanceSummaryFromDataSet( DataSet
                 dataSetQueryHelper.getColumnStringValue( dataSet, COLUMN_PROCESS_VERSION, i ),
                 dataSetQueryHelper.getColumnIntValue( dataSet, COLUMN_STATUS, i ),
                 dataSetQueryHelper.getColumnDateValue( dataSet, COLUMN_START, i ),
+                dataSetQueryHelper.getColumnDateValue( dataSet, COLUMN_END, i ),
                 dataSetQueryHelper.getColumnStringValue( dataSet, COLUMN_IDENTITY, i ),
                 dataSetQueryHelper.getColumnStringValue( dataSet, COLUMN_PROCESS_INSTANCE_DESCRIPTION, i ),
                 dataSetQueryHelper.getColumnStringValue( dataSet, COLUMN_CORRELATION_KEY, i ),
-                dataSetQueryHelper.getColumnLongValue( dataSet, COLUMN_PARENT_PROCESS_INSTANCE_ID, i ) );
+                dataSetQueryHelper.getColumnLongValue( dataSet, COLUMN_PARENT_PROCESS_INSTANCE_ID, i ),
+                dataSetQueryHelper.getColumnDateValue( dataSet, COLUMN_LAST_MODIFICATION_DATE, i ) );
```

Comment:
This mapping could be a good candidate to be isolated and tested to check if the model get the expected values from the dataSetQueryHelper.